### PR TITLE
add command http-request-duration

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

File: wazo_debug/http_request_duration.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            action='store',
+            required=False,
+            default='/var/log/nginx/wazo.access.log',
+            help='The Nginx access file to analyze',
+        )
+        return parser
+
+    def take_action(self, parsed_args):
+        print(f'Analyzing log file {parsed_args.access_file}')
+        command = (
```
Comment (pc-m) [inline wazo_debug/http_request_duration.py:33]: I would like to allow gz files to be checked
```diff
diff --git a/wazo_debug/http_request_duration.py b/wazo_debug/http_request_duration.py
index 408aa77..6e548fe 100644
--- a/wazo_debug/http_request_duration.py
+++ b/wazo_debug/http_request_duration.py
@@ -27,8 +27,8 @@ class HTTPRequestDurationCommand(Command):
     def take_action(self, parsed_args):
         print(f'Analyzing log file {parsed_args.access_file}')
         command = (
-            '''awk -F' ' '{ print $(NF-1) " " $0 }' '''
-            + parsed_args.access_file
+            '''zgrep '' ''' + parsed_args.access_file
+            + '''| awk -F' ' '{ print $(NF-1) " " $0 }' '''
             + ''' | grep -v websocket | grep -v asterisk/ws | sort -n | less +G'''
         )
         try:
```

Comment (pc-m) [reply wazo_debug/http_request_duration.py:33]: The `zgrep '' <filename>` behaves like a `cat` or `zcat` without having to check the file type

Comment (sduthil) [reply wazo_debug/http_request_duration.py:33]: Great idea, thanks :)
I used `zcat -f` instead of `zgrep`

Comment (pc-m) [reply wazo_debug/http_request_duration.py:33]: Cool, I didn't know about `zcat -f` thanks

**Review by pc-m — APPROVED** [review]

**Comment by pc-m** [comment]
recheck

**Comment by sduthil** [comment]
regate

**Comment by sduthil** [comment]
recheck

**Pull request merged**